app/workers/follows/send_email_notification_worker.rb

```diff
@@ -0,0 +1,17 @@
+module Follows
+  class SendEmailNotificationWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :mailers, retry: 10
+
+    def perform(follow_id, mailer = NotifyMailer.name)
+      follow = ::Follow.find_by(id: follow_id, followable_type: "User")
```

Comment:
Did you have namespacing issues? the double colons shouldn't be needed :)

Reply:
Yeah, got used to that since before rails 6 the autoloader sometimes has random hiccups when a class from a different module with the same name was loaded before. It is indeed pointless here. Will remove it.